ext/http/00_serve.ts

```diff
@@ -584,8 +584,11 @@ function mapToCallback(context, callback, onError) {
         if (METRICS_ENABLED) {
           op_http_metric_handle_otel_error(req);
         }
-        // deno-lint-ignore no-console
-        console.error("Exception in onError while handling exception", error);
+        import.meta.log(
+          "error",
+          "Exception in onError while handling exception",
+          error,
+        );
```

Comment:
Can you do these `import.meta.log` changes in a different commit than the `deno_core` upgrade in case we need to revert?